test/data/generate-histograms.py

```diff
@@ -0,0 +1,51 @@
+from collections import Counter
```

Comment:
I think it is totally justified to use a Python script here. But could you maybe make it adhere to [PEP 8](https://www.python.org/dev/peps/pep-0008/)?

Also, I see that you are using `rasterio`. This is fine, but maybe add a `requirements.txt` and a small Readme how to use that script?